shuup/admin/modules/sales_dashboard/dashboard.py

```diff
@@ -48,7 +47,7 @@ def get_chart(self):
         bar_chart.add_data(
             _("Sales (%(currency)s)") % {"currency": self.currency},
             [
-                bankers_round(v["sum"], 2)  # TODO: To be fixed in SHUUP-1912
+                v["sum"]
                 for v in aggregate_data.values()
```

Comment:
now that the 1st line of this is way shorter, you could maybe use different notation `[v["sum"] for v in aggregate_data.values]` instead of multiple lines?